# Computational and Systems Biologist

### Technical Skills: C++, R, MATLAB, Python

### Biography

I am a Postdoctoral Researcher at the Wisconsin Institute for Discovery, University of Wisconsin–Madison, working in the Roy Lab under the mentorship of [Professor Sushmita Roy](https://biostat.wiscweb.wisc.edu/staff/roy-sushmita/). My research focuses on the development and application of machine learning and network-based methods for analyzing single-cell and bulk multi-omics data, with particular emphasis on gene regulatory networks and evolutionary systems biology in both mammalian and plant systems. <br>

I completed my Ph.D. in Systems Biology from the University of Calcutta, where I investigated neurophysiopharmacological mechanisms of human brain networks in health and disease using integrative computational approaches. My research was jointly supervised by [Professor Nilkanta Chakrabarti](https://www.caluniv.ac.in/academic/CPEPA/Nilkanta-Chakrabarti.pdf), a distinguished neurophysiologist from the Department of Physiology, University of Calcutta, who provided critical insights into electrophysiology and translational neuroscience, and [Professor Basant K. Tiwary](https://www.pondiuni.edu.in/faculy_profiles/dr-basant-kumar-tiwary/), a leading bioinformatician from the Department of Bioinformatics, Pondicherry University, who guided the computational modeling and systems biology components. My doctoral work was supported by the DBT-BINC Fellowship (All-India Rank 17), awarded by the Department of Biotechnology, Government of India. <br>

Prior to that, I earned my Master’s degree in Pharmacoinformatics from NIPER Mohali, where I worked under the mentorship of [Professor Prasad Bharatam](https://en.wikipedia.org/wiki/Prasad_V._Bharatam), a respected expert in computational drug design, focusing on the structural characterization of antileishmanial targets. I completed my undergraduate studies with a Bachelor of Pharmaceutical Technology from Jadavpur University, where I developed a strong foundation in pharmaceutical sciences. <br>

My research background combines bioinformatics, network biology, computational pharmacology, and artificial intelligence, aiming to bridge data-driven discoveries with translational relevance. <br>

My work has been published in journals such as Scientific Reports and Life Sciences, with multiple presentations at international conferences in computational biology and neuroscience. I have received several honors, including best poster awards and recognition at symposia hosted by ISCB and the Wisconsin Stem Cell Symposium. <br>

I am also an active scientific reviewer for journals and conferences, including IEEE/ACM Transactions on Computational Biology and Bioinformatics, WIREs DMKD, and RECOMB, and I hold membership in the International Society for Computational Biology (ISCB).  <br>

Beyond research, I am passionate about science communication, open-source development, and mentoring students in bioinformatics and systems biology.  <br>

### Education
Post-Doc | Wisconsin Institute for Discovery, University of Wisconsin-Madison (2023-) <br>
Ph.D. (Science), Psychophysiology And Computation (Psychology/Physiology/Applied Mathematics) | University of Calcutta (2017-2023) <br>
M.S. (Pharm.), Pharmacoinformatics | National Institute of Pharmaceutical Education & Research (NIPER) SAS Nagar (2014-2016) <br>
B.Pharm., Pharmaceutical Technology | Jadavpur University (2010-2014) <br>

### Ph.D. Thesis
[A Systems Biology Approach for Neurophysiopharmacological Analysis of Human Brain Networks in Health and Disease](https://shodhganga.inflibnet.ac.in/handle/10603/590100)

### Publication List
1. **Hazra S**, Chaudhuri AG, Tiwary BK, Chakrabarti N. Matrix metallopeptidase 9 as a host protein target of chloroquine and melatonin for immunoregulation in COVID-19: A network-based meta-analysis. _Life Sci_. 2020;257:118096. doi:[10.1016/j.lfs.2020.118096](https://doi.org/10.1016/j.lfs.2020.118096) <br>
2. **Hazra S**, Chaudhuri AG, Tiwary BK, Chakrabarti N. Integrated network-based multiple computational analyses for identification of co-expressed candidate genes associated with neurological manifestations of COVID-19. _Sci Rep_. 2022;12(1):17141. doi:[10.1038/s41598-022-21109-3](https://doi.org/10.1038/s41598-022-21109-3) <br>
