### Biography

My research develops and applies machine learning and network-based methods to analyze single-cell and bulk multi-omics data, focusing on genomic and genetic mechanisms in development, disease, and evolution. My long-term goal is to lead an independent, interdisciplinary research program that creates scalable, interpretable computational frameworks to identify genetic determinants, genomic programs, and the system-level organization of complex biological phenotypes. I hold a Ph.D. in Systems Biology from the University of Calcutta, specializing in neurophysiopharmacological analysis of the human brain in health and disease. My work combines bioinformatics, network biology, and computational genomics to generate data-driven insights into complex biological systems.  <br><br>

### Career Objectives

My objective is to integrate machine learning–driven multi-omics analysis with systems biology in order to identify genetic mechanisms and therapeutic targets relevant to disease, development, and evolution. I seek to bridge computational discovery with experimental validation. <br><br>

### Education
Post-Doc | Wisconsin Institute for Discovery, University of Wisconsin-Madison (2023-Present) <br>
Ph.D. (Science), Psychophysiology And Computation (Psychology/Physiology/Applied Mathematics) | University of Calcutta (2017-2023) | Ph.D. Thesis: [A Systems Biology Approach for Neurophysiopharmacological Analysis of Human Brain Networks in Health and Disease](https://shodhganga.inflibnet.ac.in/handle/10603/590100) <br>
M.S. (Pharm.), Pharmacoinformatics | National Institute of Pharmaceutical Education & Research (NIPER) SAS Nagar (2014-2016) <br>
B.Pharm., Pharmaceutical Technology | Jadavpur University (2010-2014) <br><br>

### Computational Skills

• _**Programming:**_ Bash, C++, R, Python, MATLAB <br>
• _**Omics:**_ Seurat, Scanpy, scVI, LIGER, ArchR <br>
• _**Molecular Modeling:**_ PyMOL, AutoDock, AMBER, Modeller <br>
• _**Network Biology:**_ Cytoscape, Enrichr, GO <br>
• _**Office Tools:**_ MS-Excel, MS-Powepoint, MS-Word, Notepad <br>
• _**Operating systems:**_ MacOS, Windows, Linux, Android <br><br>


### Research outputs at a glance<br><br>

### Paradigm-1: Systems Biology and Network Modeling to Identify Therapeutic Targets in Human Disease<br><br>

Using systems biology and network modeling to integrate multi-layer genomic data, uncover co-expression modules, and identify candidate disease genes for therapeutic targeting. <br><br>

#### **Systems Biology Network Modeling for Drug Repurposing in Infectious Disease**<br>

<img width="385" height="312" alt="image" src="https://github.com/user-attachments/assets/a1407d99-3712-4083-8b60-c3a5f64d4dec" /> <br>
The [study](https://doi.org/10.1016/j.lfs.2020.118096) revealed a mechanistic role for MMP9 in regulating inflammatory and immune responses during SARS-CoV-2 infection and provided a systems biology rationale for host-directed therapeutic strategies and drug repurposing in COVID-19.<br>


#### **Network Analysis Reveals Candidate Coexpressed Genes for COVID-19–Associated Neurological Effects**<br>

<img width="446" height="350" alt="image" src="https://github.com/user-attachments/assets/07f11ef0-9557-4179-a19a-161d839652fe" /> <br>

An integrated literature- and database-driven framework, combined with transcriptome analyses, revealed co-expressed gene networks associated with neurological manifestations of COVID-19. The [study](https://www.nature.com/articles/s41598-022-21109-3) provides a systems-level view of SARS-CoV-2–driven neurobiological effects and highlights potential biomarkers and therapeutic targets.


### Paradigm-2: Comparative Regulatory Genomics Across Species<br><br>


A central question in evolutionary biology is how gene regulatory programs diverge across species to generate phenotypic diversity. Using Phylogenetic Dynamic Regulatory Module Networks (PDRMN), which integrates cross-species multi-omics data (RNA-seq, ATAC-seq, ChIP-seq) with phylogenetic relationships, gene regulatory programs were inferred at the module level for five mammalian species. This [analysis](https://youtu.be/sNHxvUtBWGc?si=ajokA9orU_5Zgv-o) revealed conserved and lineage-specific transcriptional regulators, evolutionary divergence in enhancer–promoter interactions, and long-term shifts in cardiovascular regulatory networks over ~100 million years.


### Paradigm-3: Modeling Dynamic Gene Regulatory Networks from single-cell Multi-Omics<br><br>

Using single-cell RNA-seq and ATAC-seq data to understand how gene regulatory networks change during dynamic processes during development and disease.









